website/public/locales/en/tasks.json

```diff
@@ -20,7 +20,7 @@
   },
   "reply_as_user": {
     "label": "Reply as User",
-    "desc": "Chat with Open Assistant and help improve its responses as you interact with it.",
+    "desc": "Chat with Open Assistant and help improve it's responses as you interact with it.",
```

Comment:
https://github.com/LAION-AI/Open-Assistant/pull/2037 made this change. As I'm not a native speaker, I'm not sure which is correct

Reply:
it's a typo, sorry.